data/helpers.d/apt

```diff
@@ -218,6 +219,20 @@ ynh_install_app_dependencies () {
     fi
     local dep_app=${app//_/-}	# Replace all '_' by '-'
 
+    # Handle specific versions
+    if [[ "$dependencies" =~ [\<=\>] ]]
+    then
+        # Replace version specifications by relationships syntax
+        # https://www.debian.org/doc/debian-policy/ch-relationships.html
+        # Sed clarification
+        # [^(\<=\>] ignore if it begins by ( or < = >. To not apply twice.
+        # [\<=\>] matches < = or >
+        # \+ matches one or more occurence of the previous characters, for >= or >>.
+        # [^,]\+ matches all characters except ','
+        # Ex: 'package>=1.0' will be replaced by 'package (>= 1.0)'
+        dependencies="$(echo "$dependencies" | sed 's/\([^(\<=\>]\)\([\<=\>]\+\)\([^,]\+\)/\1 (\2 \3)/g')"
```

Comment:
![myeyes](https://user-images.githubusercontent.com/4533074/78511138-ad09f000-779a-11ea-9eab-25b3abac5cac.jpeg)

Reply:
Agree, it burns...
That's why it's explained just before ;)